operations/notification.go

```diff
@@ -39,10 +40,14 @@ func notificationSlack() cli.Command {
 			},
 		},
 		Action: func(c *cli.Context) error {
-			confPath := c.Parent().String(confFlagName)
+			confPath := c.Parent().Parent().String(confFlagName)
 			target := c.String(targetFlagName)
 			msg := c.String(msgFlagName)
 
+			if err := validateTargetHasOctothorpeOrArobase(target); err != nil {
+				return errors.Wrap(err, "invalid target")
```

Comment:
Just return the error. In this case the error message is so clear that wrapping it with a stack just gives users more text to read through.

Reply:
Done.